cmd/frcli/node_audit.go

```diff
@@ -17,6 +18,36 @@ var onChainReportCommand = cli.Command{
 	Name:     "audit",
 	Category: "reporting",
 	Usage:    "Get a report of node activity.",
+	Description: `
+	Create a report containing all of your node's activity over 
+	the period specified. Transactions are sorted into a set of 
+	lightning-specific entry types. Fiat values can optionally be 
+	included using the --enable_fiat flag. To write the report 
+	directly to a csv, set a target directory using the --csv_path 
+	flag.
+
+	These reports can optionally be created with custom categories. 
+	This requires providing a name for the category, and a set of 
+	regular expressions which identify the labels of transactions 
+	belonging in the category. To directly string match, just provide
+	the string itself. These regular expressions are matched against 
+	the labels for on chain transactions and the invoices on memos 
+	(at present we cannot match forwarding events and payments).
+
+	Categories should be expressed as a json array with the 
+	following format:
+	--categories='[
+		{ 
+			"name": "test", 
+			"on_chain": true, 
+			"label_identifiers": ["swap", "test"] 
```

Comment:
Nice that we can use the RPC struct directly! Maybe we should rename the `label_identifiers` field to something like `label_patterns` or `label_imatchers` though to make it more apparent that you can specify regular expressions (in this example specifically where you don't see the RPC docs)? Or just add an example pattern that looks more like regex?

Reply:
Renamed and added a more obviously regexey example!